### ISSUE TITLE
kontena stack build NameError: undefined local variable or method `service_prefix'

### ISSUE THREAD

**Issue by SpComb** [kontena/kontena#1421]
`kontena stack build -f ~/kontena/examples/whoami/kontena.yml`

```
NameError: undefined local variable or method `service_prefix' for #<Kontena::Cli::Stacks::BuildCommand:0x000000027c20e8>
Did you mean?  service_list
  /home/kontena/kontena/kontena/cli/lib/kontena/cli/stacks/build_command.rb:18:in `execute'
  /home/kontena/kontena/kontena/cli/lib/kontena/command.rb:187:in `run'
  /home/kontena/kontena/kontena/cli/vendor/bundle/ruby/2.3.0/gems/clamp-1.1.1/lib/clamp/subcommand/execution.rb:11:in `execute'
  /home/kontena/kontena/kontena/cli/lib/kontena/command.rb:187:in `run'
  /home/kontena/kontena/kontena/cli/vendor/bundle/ruby/2.3.0/gems/clamp-1.1.1/lib/clamp/subcommand/execution.rb:11:in `execute'
  /home/kontena/kontena/kontena/cli/lib/kontena/command.rb:187:in `run'
  /home/kontena/kontena/kontena/cli/vendor/bundle/ruby/2.3.0/gems/clamp-1.1.1/lib/clamp/command.rb:132:in `run'
  bin/kontena:16:in `<top (required)>'
```

Looks similar to #1409

**Comment by kke** [comment]
Is it supposed to ask the questions, run the extends, etc?

**Comment by jnummelin** [comment]
Fixed in #1434
